ciapp/src/test/java/com/group7/ciapp/ProjectTest.java

```diff
@@ -34,7 +34,10 @@ void testCloneRepo_true() {
     }
 
     /**
-     * Clone the repo and compare it with an non-existing path. The original path should not be
+     * Clone the repo and compare it with an non-existing path. The original path
+     * 
```

Comment:
Maybe remove this and the one below? Formatting looks a bit off for this comment.